src/coreclr/System.Private.CoreLib/src/System/RuntimeType.CoreCLR.cs

```diff
@@ -2757,7 +2757,12 @@ public override InterfaceMapping GetInterfaceMap([DynamicallyAccessedMembers(Dyn
                 MethodBase? rtTypeMethodBase = GetMethodBase(reflectedType, classRtMethodHandle);
                 // a class may not implement all the methods of an interface (abstract class) so null is a valid value
                 Debug.Assert(rtTypeMethodBase is null || rtTypeMethodBase is RuntimeMethodInfo);
-                im.TargetMethods[i] = (MethodInfo)rtTypeMethodBase!;
+                MethodInfo targetMethod = (MethodInfo)rtTypeMethodBase!;
+                // https://github.com/dotnet/runtime/issues/90863
```

Comment:
```suggestion
                // TargetMethods array should only contain generic method definitions
```
Comments are better than links to issue